packages/web-components/src/radio-group/README.md

```diff
@@ -0,0 +1,127 @@
+# Radio Group
+
+> RadioGroup lets people select a single option from two or more Radio items. Use RadioGroup to present all available choices if there's enough space. For more than 5 choices, consider using a different component such as Dropdown.
```

Comment:
Dropdown may not exist here, let's update this to not suggest using components we don't have or know will exist :).